docs/data/data-grid/filtering/header-filters.md

```diff
@@ -49,7 +49,7 @@ The following demo uses the `renderHeaderFilter` method to customize the header
 
 ### Customize using `filterOperators`
 
-If the filter operator has a [custom `InputComponent`](https://mui.com/x/react-data-grid/filtering/#custom-input-component), the same component is being used for the header filter.
+If the filter operator has a [custom `InputComponent`](/x/react-data-grid/filtering/customization/#custom-input-component), the same component is being used for the header filter.
```

Comment:
That link was broken.

Reply:
Wasn't caught by the `link-check` because of the full URL I guess?

Reply:
Correct